src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,82 @@
+@import "../utils/placeholders";
+
+.product-card {
+  padding: 29px 14px 14px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: 400ms;
+
+  &:hover {
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  }
+  &__image {
+    display: block;
+    margin: 0 auto 45px;
+    width: 160px;
+    height: 134px;
+    transition: transform 400ms;
+
+    &:hover {
+      transform: scale(1.2);
+      border: 3px solid #f3f3f3;
+    }
+  }
+  &__text {
+    margin-bottom: 6px;
+    font-weight: 500;
+    font-size: 12px;
+    background-color: #fff;
+    line-height: 15px;
+  }
+  &__product-code {
+    width: 100%;
+    @extend %small-text;
+    margin-bottom: 15px;
+  }
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 20px;
+  }
+  &__rewiews {
+    display: inline-block;
+    @extend %small-text;
+  }
+  &__price {
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 20px;
+  }
+  &__price-text {
+    margin: 0;
+    @extend %small-text;
+  }
+  &__price-value {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+  }
+  &__buy-link {
+    display: flex;
+    justify-content: center;
+    padding: 11px 0;
+    width: 100%;
+    background-color: $blue-color;
+    border-radius: 5px;
+    color: white;
+    font-weight: 700;
+    font-size: 14px;
+    text-decoration: none;
+    border: 1px solid $blue-color;
+    transition: 400ms;
```

Comment:
and here

Reply:
If we are didn`t touch additional task - it`s ok!